### PR TITLE
Feat/change articulos medium

### LINKED ISSUE #196: Change "Articulos de medium" for "Artículos de blog"
Include articles in Spanish and English from website and Medium

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### GH Issue

Resolves #196  

### Steps to test
1. Check title "Artículos de Medium" in the home guide page is changed to Artículos de posts

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]
LGTM

File: src/pages/index.js [hunk 1]
```diff
@@ -254,7 +254,7 @@ const CustomArticle= (props) => {
 const MediumSection = () => {
   return (
     <div className="container section-container">
-       <h1 className="section-tittle">Artículos de Medium</h1>
+       <h1 className="section-tittle">Artículos de blogs</h1>
```
Comment (xavier506) [inline src/pages/index.js:257]: Articulos de blogs o blog?

Comment (leisterfrancisco) [reply src/pages/index.js]: Blog is correct, I wrote **blogs** to have a change to commit to try to test the netlify error, but in my last commit fixed, thank you for your observation

**Review by Edgar097 — APPROVED** [review]

**Review by steph222 — APPROVED** [review]
💯

**Pull request merged**